scripts/test/index.js

```diff
@@ -0,0 +1,9 @@
+const { spawn } = require('child_process')
+const { inputDir = './source' } = require('yargs').argv
+
+const args = [
+  require.resolve('mocha/bin/_mocha'),
+  `${inputDir}/**/__tests__/*-test.js`
```

Comment:
I see in later commits it's all gone. Should this stuff live elsewhere? I wonder what the value is of a test script if each project will need to configure their own setup is all. Currently what this seems to give the user is `mocha` being pre-installed and a default test path.